lib/api/src/js/value.rs

```diff
@@ -33,7 +34,7 @@ pub enum Value {
     F64(f64),
 
     /// An `externref` value which can hold opaque data to the wasm instance itself.
-    //ExternRef(Option<ExternRef>),
+    ExternRef(Option<Arc<ExternRef>>),
```

Comment:
Why an Arc here?

Reply:
It seems to me that the `ExternRef` should be the one storing the `Arc` (if really needed) (and hidden from the user)